dockerfile-configs/common-components.yaml

```
@@ -62,7 +61,7 @@
     info: nerdctl is a Docker-compatible CLI for containerd. The root directory of the host has to be mounted under `/host`
   - name: kubectl
     # renovate: datasource=github-releases depName=kubernetes/kubernetes
-    version: v1.30.11
+    version: v1.32.4
```

```suggestion
    version: v1.31.9
```

To cover as most as possible kubernetes versions currently supported by gardener according to the version skew policy.

We can move this to `v1.32.*` once gardener supports k8s `v1.33`